### PR TITLE
RD-5782 Translated sites widget

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
## Description

This PR is a continuation of #2433 
It's purpose is to translate `sites` widget, as instructed in [Ongoing Refactors](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2148073473/Ongoing+Refactors+in+Cloudify+UI#Adding-translation-support-in-widgets)

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4511/)

## Documentation

N/A

### CODE REVIEW

**Comment by Vorbert-Kruk** [comment]
After the weekly, I've migrated away from providing `name` and `description` labels within widget definition, to be consistent with the new approach that we are taking 🎉
Change was introduced via df9921049034fe13fed6bb88742c0cfaab03ef2b

File: widgets/sites/src/SitesTable.tsx [hunk 1]
```diff
@@ -222,7 +226,9 @@ export default class SitesTable extends React.Component<SitesTableProps, SitesTa
                 {site && (
                     <>
                         <DeleteModal
-                            content={`Are you sure you want to delete the site '${site.name}'?`}
+                            content={translate('confirm.delete', {
+                                siteName: site.name
+                            })}
```
Comment (kubama) [inline widgets/sites/src/SitesTable.tsx:231]: For simplicity the whole `site` could be passed here and `name` referenced in the translation file.

Comment (Vorbert-Kruk) [reply widgets/sites/src/SitesTable.tsx:231]: Got it, nice idea

But after looking into the code, I may suggest to leave it as it is right now 
Current approach is explicitly indicating which property is being used within `en.json` file and it's easier to correlate translation and it's usage, which may be useful while working on a future adjustments 💪 

WDYT? 🤔

Comment (kubama) [reply widgets/sites/src/SitesTable.tsx:231]: OK, let's leave it then.

**Review by kubama — APPROVED** [review]

**Pull request merged**